browser/ephemeral_storage/hsts_partitioning_browsertest.cc

```diff
@@ -152,6 +152,7 @@ class HSTSPartitioningBrowserTestBase : public InProcessBrowserTest {
     bool result = false;
     partition->GetNetworkContext()->IsHSTSActiveForHost(
         host,
+        /*is_top_level_nav=*/true,
```

Comment:
tests when false?

Reply:
Added in https://github.com/brave/brave-core/pull/27172/commits/85458b34763fc3506de355c27d8c0d18ba16c755 cc: @goodov